src/data/showcases.js

```diff
@@ -1156,6 +1156,15 @@ export const Showcases = [
     source: null,
     tags: ["governance"],
   },
+  {
+    title: "Cardano Signer",
+    description:
+      "Tool to sign and verify keys and metadata for pools, wallets and governance usage.",
+    preview: require("./builder-tools/cardano-signer.png"),
+    website: "https://github.com/gitmachtl/cardano-signer",
+    source: null,
```

Comment:
```suggestion
    website: "https://github.com/gitmachtl/cardano-signer",
    source: "https://github.com/gitmachtl/cardano-signer",
```
@gitmachtl if wondering why this branch fails to build: error messages show the `source:` link is required for projects tagged `opensource`.